picard/formats/vorbis.py

```diff
@@ -136,13 +136,16 @@ def _load(self, filename):
         file = self._File(encode_filename(filename))
         file.tags = file.tags or {}
         metadata = Metadata()
+        config = get_config()
+        date_sanitize = not config.setting['disable_date_sanitize']
         for origname, values in file.tags.items():
             for value in values:
                 value = value.rstrip('\0')
                 name = origname
                 if name in {'date', 'originaldate', 'releasedate'}:
                     # YYYY-00-00 => YYYY
-                    value = sanitize_date(value)
+                    if date_sanitize:
+                        value = sanitize_date(value)
```

Comment:
Since the variable `date_sanitize` is only used in one place, is it really necessary, or should the test in Line 147 simply be `if not config.setting['disable_date_sanitize']:` similar to the way it is done later in Line 286?

Reply:
> Since the variable `date_sanitize` is only used in one place, is it really necessary, or should the test in Line 147 simply be `if not config.setting['disable_date_sanitize']:` similar to the way it is done later in Line 286?

It is better to cache config variables used in loops. Getting their values imply quite a lot of code.

Reply:
Right.  I completely missed that.  Sorry.